packages/theme/src/cli/services/open.test.ts

```diff
@@ -58,12 +58,7 @@ describe('open', () => {
 
     // Then
     expect(renderInfo).toBeCalledWith({
-      headline: [
-        'Preview information for theme my theme',
-        {
-          subdued: `(#1)`,
-        },
-      ],
+      headline: ['Preview information for theme', 'my theme', {subdued: '(#1)'}],
```

Comment:
What is the reason for having `'Preview information for theme', 'my theme'` as two items instead of a single string?